functests/1_performance/cpu_management.go

```diff
@@ -105,6 +102,16 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 	Describe("[test_id:27492][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Verification of cpu manager functionality", func() {
 		var testpod *corev1.Pod
 
+		testutils.BeforeAll(func() {
+			if discovery.Enabled() {
+				profile, err := profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
+				Expect(err).ToNot(HaveOccurred())
+				if len(*profile.Spec.CPU.Isolated) == 1 {
+					Skip("Skipping tests since there are insufficant isolated cores to create a stress pod")
```

Comment:
Sorry did not notice before, this is gonna cause only the first item of the table to skip.
You should the same trick you did in rt, setting a flag and skipping in beforeEach

Reply:
@fedepaol  done